data/json/items/gun/223.json

```diff
@@ -390,35 +382,165 @@
+    "name": { "str": "M4 Carbine" },
+    "description": "A popular carbine, long used by the US military.  Though accurate, small, and lightweight, it is infamous for its unreliability when not properly maintained. Accepts STANAG magazines.",
+    "ascii_picture": "m4a1",
+    "weight": "2880 g",
+    "volume": "3310 ml",
+    "longest_side": "758 mm",
+    "price": 240000,
+    "price_postapoc": 5000,
+    "variants": [
+      {
+        "id": "m4a1",
+        "name": { "str": "M4A1" },
+        "description": "A popular carbine, long used by the US military.  Though accurate, small, and lightweight, it is infamous for its unreliability when not properly maintained.",
+        "ascii_picture": "m4a1",
+        "weight": 1
+      },
+      {
+        "id": "m4_cqbr",
+        "name": { "str": "MK 18 CQBR" },
+        "description": "This is a shorter M4 carbine with a 10.3 inch barrel, intended for confined spaces and close quarters battle situations."
+      }
+    ],
+    "to_hit": -1,
+    "bashing": 12,
+    "material": [ "steel", "plastic" ],
+    "symbol": "(",
+    "color": "dark_gray",
+    "ammo": [ "223" ],
+    "ranged_damage": { "damage_type": "bullet", "amount": -2 },
+    "dispersion": 180,
+    "durability": 6,
+    "min_cycle_recoil": 1350,
+    "modes": [ [ "DEFAULT", "semi-auto", 1 ], [ "AUTO", "auto", 4 ] ],
+    "pocket_data": [
+      {
+        "pocket_type": "MAGAZINE_WELL",
+        "holster": true,
+        "max_contains_volume": "20 L",
+        "max_contains_weight": "20 kg",
+        "item_restriction": [
+          "stanag30",
+          "stanag5",
+          "stanag10",
+          "stanag20",
+          "stanag40",
+          "stanag50",
+          "stanag60",
+          "stanag60drum",
+          "stanag90",
+          "stanag100",
+          "stanag100drum",
+          "stanag150",
+          "survivor223mag"
+        ]
+      }
+    ]
+  },
+  {
+    "//": "Changed from 'NATO Burst Rifle', as I found that to be too vague.",
     "id": "m16a4",
-    "copy-from": "nato_assault_rifle",
+    "copy-from": "rifle_semi",
     "looks_like": "ar15",
     "type": "GUN",
-    "name": { "str": "NATO burst rifle" },
-    "description": "An assault rifle used by a NATO member state.  Unlike other assault rifles, this one does not have an auto-fire mode, and instead has a burst fire mode.  As per NATO's STANAG agreement, it is chambered in 5.56x45mm.",
+    "name": { "str": "M16 Burst Rifle" },
+    "description": "The M16 is a very common assault rifle descended from the AR-15, used by militaries across the world for over 50 years.  It fires in three round bursts. This ensures the rifle is more accurate, but at the cost of being slower than other assault rifles.",
+    "weight": "3260 g",
+    "volume": "5030 ml",
+    "longest_side": "1003 mm",
+    "price": 240000,
+    "price_postapoc": 5000,
     "variants": [
       {
         "id": "m16a4",
-        "name": { "str": "M16A4" },
+        "name": { "str": "m16a4" },
```

Comment:
Is the lowercase intended in the **name** field?

Reply:
No, it wasn't, fixed it now. Sorry about that